index.js

```diff
@@ -49,17 +49,24 @@ module.exports = (config) => {
         });
     }
 
-    /**
+     /**
      * Checks that the provided user is allowed to perform the action described by the request
-     *
-     * @param {object} req - The request.  Compatible with expressjs request.
-     *                       Required properties: method, path
-     * @param {string} username - The subject (or username) of the requester.
+     * This request is decoupled from the request path, allowing for an alternative description of resources
+     * 
+     * @param {object} abacRequest - The Attributes Based Access Control request.  
+     *                       Required properties: subject, resource, action 
      * @returns {promise} - A promise to authorize the user.
      *                      Resolves with the user and resource attributes.
      *                      Rejected if not authorized, or an error occurs.
      */
-    acs_utils.isAuthorized = (req, username) => {
+    acs_utils.isAuthorizedFor = (abacRequest) => {
+        if (abacRequest === null) {
```

Comment:
Should also check for undefined.  This can be shortcut with just ``if(!abacRequest) {``